hudi-common/src/main/java/org/apache/hudi/common/model/HoodieRecordMerger.java

```diff
@@ -24,13 +24,12 @@
 import org.apache.hudi.common.config.TypedProperties;
 import org.apache.hudi.common.engine.RecordContext;
 import org.apache.hudi.common.model.HoodieRecord.HoodieRecordType;
+import org.apache.hudi.common.schema.HoodieSchema;
 import org.apache.hudi.common.table.HoodieTableConfig;
 import org.apache.hudi.common.table.HoodieTableVersion;
 import org.apache.hudi.common.table.read.BufferedRecord;
 import org.apache.hudi.common.util.Option;
 
-import org.apache.avro.Schema;
```

Comment:
Curious to see how many classes still import `org.apache.avro.Schema` after quite a few refactoring PRs

Reply:
Quite a lot of refactoring, reminding me of the changes introducing the `HoodieStorage` abstraction :)